### PR TITLE
desktop design

### PR DESCRIPTION
**Pull request by Obimbo07**
This branch introduces the desktop design of our portfolio page.

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @Obimbo07,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failed linters, kindly ensure all linters pass.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: - [ ] The two links in your prerequites section of your readme are currently not leading anywhere as they are broken and showing a 404 error, please kindly fix it by putting the appropriate link so your readme looks professional :+1:

Comment (Efiamotu-1) [inline index.html:1]: - [ ] Your design for the recent works section is distinctively different from the figma template and i have drawn lines in the image below to show you the places where you have a different design compared to what you should have on your portfolio. 
- [ ]  The text and hr line for recent works should be side by side on the desktop version and not one ontop of the other, please kindly fix this design irregularity.
- [ ] Your image for this section is not spanning more than half of the width of the viewport like the figma design as the text takes up much more of the space. Both contents should be displayed like the figma design below and centered in the middle with equal amount of space to the left and right of the containing element. as seen bt the red lines on the figma design. 
- you can achieve this by giving the containing div a fixed width and then centering it in the middle using `margin:auto` :+1: 
- figma design
r-w
- your design
r-w-s

Comment (Efiamotu-1) [inline index.html:1]: - [ ] on hover of every card as shown by the first card in the figma template below the `see project` button should e displayed, however this is not the current design of each of your cards, as all of the contents don't display and then the `see project` button is not visible 
- please ensure to work on designing your portfolio website as correctly seen on figma before requesting for a new review :+1: :grin: 
- figma design
h-f

Comment (Efiamotu-1) [inline index.html:1]: - [ ] From the figma template below your technologies section should have an horizontal line but it is not required to span across the full width of the page but rather centered in the middle same for the contents of `language, framework & skills`
- you can achieve this by giving the containing div a fixed width and then centering it in the middle using `margin:auto` :+1:
- figma design
a-f

Comment (Efiamotu-1) [inline index.html:1]: - [ ] please you can see from the below lines drawn on the figma image and your image you are not designing the contact me section and footer correctly as provided by the figma design please make sure to achieve the figma design as much as possible
- [ ] It is important to make all of the required corrections pertaining to the design structure of your webpage as correctly designing your portfolio to look like the figma design shows how much attention to details you pay as a developer and this is a good skills recruiters and clients are looking out for :+1: :sparkles: 
- figma design
f-f
- your design
f-s

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Obimbo07 , 



Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 1]
```diff
@@ -455,3 +460,220 @@ fieldset {
...
+  }
+
+  .card-works:hover h2,
+  .card-works:hover p {
+    color: transparent;
+  }
+
+  .card-works:hover ul {
+    display: none !important;
+  }
```
Comment (AbelG101) [inline styles.css]: - [ ] Ouch! this is close but it seems like the hover effect on your project cards is not functioning as shown in the Figma design
    - Figma design:
      image
    - As you can see from the above design when the mouse is hovered over the project card the project image should be displayed with the "See project" button at the bottom taking the full width. 
    - Your design:
      firefox_rJpUlmejoC
     - In your design, the project image is not displayed, and the button doesn't take the full width of the card so please work on that ✔

File: styles.css [hunk 2]
```diff
@@ -455,3 +460,220 @@ fieldset {
...
+  .bottom-footer {
+    display: flex;
+    flex-direction: row;
+    gap: 0;
+  }
+
+  .contact-section {
+    height: auto;
+    width: 445px;
+  }
```
Comment (AbelG101) [inline styles.css]: - [ ] As mentioned previously please make sure your contact form looks similar to the given Figma design.
    - Your design:
      Inkedfirefox_NpRxUyp93R
    - Figma design:
      image
- [ ] As you can see from your design you seem to be missing some spacing to the left of your text, you can fix that by adding a `margin-left: 6rem;` 😉

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Obimbo07 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @Obimbo07 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Changes Implemented :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**